.github/local-actions/branch-manager/lib/main.ts

```diff
@@ -77,8 +77,14 @@ async function main(repo: {owner: string; repo: string}, token: string, pr: numb
     {git, config},
     pr,
     PullRequestValidationConfig.create({
-      assertPending: false,
+      assertPending: true,
+      assertSignedCla: true,
```

Comment:
Wasn't there a resource issue if we process non-merge ready PRs?

Reply:
Whoops, yes we need to keep merge ready as true.